pubtools/pulplib/__init__.py

```diff
@@ -12,6 +12,7 @@
     Unit,
     FileUnit,
     RpmUnit,
+    Dependency,
```

Comment:
Could this class please be named in a way which shows it's RPM-specific?  e.g. RpmDependency?

The concern is that other types of data can also have dependencies and we may need to support those some day, but this Dependency class wouldn't be usable for them.